src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2935,7 +2935,7 @@ void processContextElement
   std::map<std::string, std::string>&  uriParams,   // FIXME P7: we need this to implement "restriction-based" filters
   const std::string&                   xauthToken,
   const std::string&                   apiVersion,
-  bool                                 checkEntityExistance
+  const std::string&                   ngsiv2Flavour
```

Comment:
Maybe we can can add is a FIXME message on Ngiv2Flavour enum about the possibility of using Request enum in the future, depending on how the flavour things evolved in the future.